src/trigger/text_area.cpp

```diff
@@ -149,6 +159,8 @@ TextArea::get_settings()
   settings.add_bool(_("Once"), &m_once, "once");
   settings.add_float(_("Text change time"), &m_delay, "delay");
   settings.add_float(_("Fade time"), &m_fade_delay, "fade-delay");
+  settings.add_string_select(_("Anchor"), &m_anchor,
+    { _("Middle"), _("Top-left"), _("Top"), _("Top-right"), _("Left"), _("Right"), _("Bottom-left"), _("Bottom"), _("Bottom-right") }, 0, "anchor-point");
```

Comment:
`add_enum` would be better adapted for this, since it refers to a logically ordered enum; you may find an example in `src/objects/custom_particle_system.cpp:429`. It helps avoid having to declare `anchor_points`, lets you use the `AnchorPoint` enum as a type for `m_anchor`, and removes the need to call `string_to_anchor_point`.

I'll mark the other areas for clarity :smile:

Reply:
I was trying to do like this but AnchorPoint uses bitmasks so it is not ordered enum

Reply:
It should be fixed though; the current method has many flaws.

I can take care of this in another PR when I'll have some time.

Reply:
I will try to come up with different method then